src/HW6.java

```diff
@@ -0,0 +1,30 @@
+import java.util.*;
+public class HW6 {
+    public static void main(String[] args) {
+        // task 1
+        int[] array1 = new int[]{5, 7, 11, 20};
+        Arrays.sort(array1);
+        int minDiff = array1[1] - array1[0];
+        for (int i = 2; i != array1.length; i++) {
+            minDiff = Math.min(minDiff, array1[i] - array1[i - 1]);
+        }
+        System.out.println("The minimal difference is " + minDiff);
+
+        //task 2
```

Comment:
Please try to exchange repeated actions with loop. Avoid hardcoding :)
Create a method with implementation and then use the method in the main method